src/ZkMinterModTriggerV1.sol

```diff
@@ -85,4 +110,16 @@ contract ZkMinterModTriggerV1 is ZkMinterV1 {
 
     emit TriggerExecuted(msg.sender, targets.length);
   }
+
+  /// @notice Sends minted tokens held by this contract to the immutable recovery address.
+  /// @param _amount The amount of tokens to send.
+  /// @dev Only callable by addresses with the DEFAULT_ADMIN_ROLE.
+  function recoverTokens(address _token, uint256 _amount) external {
+    _revertIfClosed();
```

Comment:
This brings up some good questions should close, recover any token balances?

Reply:
To recover tokens when closing we'd have to create a new `close` function that takes a token address as an argument, while there would still be a `close()` function that's inherited from `ZkMinterV1` that doesn't recover any tokens. 

On a separate note, should we make all the functions virtual so the V2 version is easier to iterate on?

Reply:
Now allowing token recovery while paused or closed https://github.com/zksync-association/zkminters/commit/028c9658456c241c99b39b2034ba4d3502c5ae7a